ocaml/xapi/dbsync_master.ml

```diff
@@ -336,6 +336,11 @@ let setup_telemetry ~__context =
       )
       ()
 
+let update_pool_recommendations ~__context =
+  let pool = Helpers.get_pool ~__context in
```

Comment:
Please wrap this with `Helpers.log_exn_continue` like the other functions above. We don't want this to raise exceptions for any reason.